Makefile

```diff
@@ -36,7 +36,7 @@ export GO111MODULE := on
 # Kubebuilder.
 #
 #To do check kubernetes version
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.20.1
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.23.3
```

Comment:
This is because, the older version 1.20.1 is no longer maintained and curl would return 404.